src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,179 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    private static final String NAME = "duke.txt";
+    private static final String DIRECTORY = "data";
+
+    public static void main(String[] args) throws IOException {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        int taskCount = 0;
+
+        printGreeting();
+        FileManager file = new FileManager(NAME, DIRECTORY);
+        try {
+            taskCount = file.loadSavedFile(list, taskCount);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
Should there be a more descriptive message besides printing out the stack trace? Perhaps you can think of the scenarios where it will throw this exception.